shared/services/config/lighthouse-config.go

```diff
@@ -136,21 +133,3 @@ func (cfg *LighthouseConfig) GetName() string {
 func (cfg *LighthouseConfig) GetConfigTitle() string {
 	return cfg.Title
 }
-
-// Get the appropriate LH default tag for production
-func getLighthouseTagProd() string {
-	missingFeatures := sys.GetMissingModernCpuFeatures()
-	if len(missingFeatures) > 0 {
-		return lighthouseTagPortableProd
-	}
-	return lighthouseTagModernProd
-}
-
-// Get the appropriate LH default tag for testnets
-func getLighthouseTagTest() string {
-	missingFeatures := sys.GetMissingModernCpuFeatures()
```

Comment:
Let's remove `GetMissingModernCpuFeatures()` as well. It's only used here, and to complain when someone sets -modern but isn't. (`rocketpool-cli/service/service.go:checkCpuFeatures`)